astroplan/plots/mplstyles.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.visualization import astropy_mpl_style
+
+__all__ = ['light_style_sheet', 'dark_style_sheet',
+           'available_style_sheets']
+
+available_styles = ["light_style_sheet", "dark_style_sheet"]
+
+def available_style_sheets():
```

Comment:
Is there a way to reference this variable in the sphinx docs? When I referenced `~astroplan.plots.available_styles` it was throwing errors.